### DISCUSSION TITLE
Unknown profiler got from ArgoIndex(index_file='bgc-s').load()

### DISCUSSION THREAD

**Discussion by cywhale** [euroargodev/argopy#419] [Q&A]
```
from argopy import DataFetcher, ArgoNVSReferenceTables, ArgoIndex
import argopy
sidx = ArgoIndex(index_file='bgc-s').load() 
dfs = sidx.to_dataframe()
dfs[["profiler_code", "profiler"]]
```
Got all Unknown in profiler column:
```
profiler_code | profiler
-- | --
846 | Unknown
846 | Unknown
846 | Unknown
846 | Unknown
846 | Unknown
... | ...
834 | Unknown

```
I'm not sure if profiler_code and profiler comes from R08 table:
```
test_profiler = ArgoNVSReferenceTables().tbl('R08')
print(test_profiler)
```
Got:
   altLabel                                          prefLabel  \
0       890                            PROVOR_III - Jumbo, SBE   
1       891                            PROVOR_III - Jumbo, RBR   
2       841          PROVOR float with SBE conductivity sensor   
3       838    ARVOR-D deep float with SBE conductivity sensor   
4       831                                       PALACE float   
5       889                              PROVOR_V - Jumbo, RBR   

If it's profiler in this table, then it should have values, not to be Unknown.
I had tried argopy==0.1.17 and 1.0.0, both got the same results. But I'm sure this dataframe have not-unknown values in profiler column maybe several months ago at last time I checked it.
Is there something wrong in my code as I using argopy? Thanks.

**Comment by gmaze** [comment]
Hi @cywhale 
thanks for reporting this !

After some investigation, indeed there's bug in the reference table dictionary mapping,
I created an issue with this in here #420 
Fixing this today !
